src/main/scala/org/renci/chemotext/Main.scala

```diff
@@ -73,7 +76,7 @@ object Main extends App with LazyLogging {
       (meshSubstanceIDs, meshSubstanceLabels) = (article \\ "NameOfSubstance").map(substance => ((substance \ "@UI").text, substance.text)).unzip
       allMeshTermIDs = meshTermIDs.flatten.toSet ++ meshSubstanceIDs
       allMeshLabels = meshLabels.toSet ++ meshSubstanceLabels
-    } yield (pmid, s"$title $abstractText ${allMeshLabels.mkString(" ")} $geneSymbols", allMeshTermIDs)
+    } yield ArticleInfo(pmid, s"$title $date $abstractText ${allMeshLabels.mkString(" ")} $geneSymbols", allMeshTermIDs)
```

Comment:
If the date is passed as a parameter for the ArticleInfo (suggested above), I would not include it in this text, which is the string passed to the entity recognizer.